### PR TITLE
Added Link style to buttons for mac

### PR DESCRIPTION
**Pull request by Abhityagi16**
Fixed Issue #80

### CODE REVIEW

File: DiceKeys/Extensions/Button.swift [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+//  Button.swift
+//  DiceKeys
+//
+//  Created by Abhishek Tyagi on 02/02/21.
+//
+
+import SwiftUI
+
+extension Button {
+
```
Comment (UppaJung) [inline DiceKeys/Extensions/Button.swift:11]: Is this still needed?

File: DiceKeys/Extensions/View.swift [hunk 2]
```diff
@@ -57,4 +57,10 @@ extension View {
         // iphone 7: 667 × 375
         self.previewLayout(PreviewLayout.fixed(width: 320, height: 568))
     }
+    
+    @ViewBuilder func applyCustomStyle() -> some View {
```
Comment (UppaJung) [inline DiceKeys/Extensions/View.swift:61]: Name should indicate what this is for (buttons) and why we need it.  How about `applyLinkButtonStyleForEveryOS`

**Review by UppaJung — CHANGES_REQUESTED** [review]

File: DiceKeys/Extensions/Button.swift (hunk 1 above)
Comment (Abhityagi16) [reply DiceKeys/Extensions/Button.swift:11]: Xcode is a pain, I removed this file from xcode. No idea how it got here

**Review by UppaJung — APPROVED** [review]
Thanks so much for the quick fixes.

**Pull request merged**